src/panels/ImageDropdownUI.tsx

```diff
@@ -58,6 +59,16 @@ export const ImageDropdownMenuUI = observer(function ImageDropdownMenuUI_(p: { i
             </MenuItem>
 
             <div className='divider my-1'>FileSystem</div>
+            <MenuItem
+                icon={<IkonOf name='mdiStarShooting' color='gold' />}
```

Comment:
just for the record; when merging into my dev branch, icons are now specified this way
```tsx
            <MenuItem
                icon='mdiStarShooting' <----
                iconClassName='[color:gold]' <----
```